changelog.json

```
@@ -1,5 +1,13 @@
 {
   "releases": {
+    "2.6.4-beta1": [
+      "[Added] Add branch context menu with rename and delete to branch dropdown list - #5803 #10432",
+      "[Fixed] Always respect the default branch name chosen by the user - #11447",
+      "[Fixed] Notepad++ is not closed anymore with GitHub Desktop - #11518",
+      "[Improved] Suggest email addresses from GitHub accounts and warn about misattributed commits during the onboarding and in Preferences - #11561 #11566",
```

Just trying to be a bit more concise here, nothing substantive.
```suggestion
      "[Improved] Suggest emails from GitHub accounts and warn about misattributed commits in onboarding and Preferences - #11561 #11566",
```